backend/src/main/java/com/bakdata/conquery/resources/admin/rest/AdminProcessor.java

```diff
@@ -128,7 +130,18 @@ else if (!table.getDataset().equals(dataset)) {
 	}
 
 	public void addConcept(Dataset dataset, Concept<?> c) {
+		// Add the standard EventDateUnionSelect to every added Concept
```

Comment:
hast du eigentlich überprüft ob das auch so bei den slaves ankommt, oder ob die das dann sogar doppelt haben?

Reply:
Bin ehrlich gesagt kein Fan von, mir wäre es lieber wir würden die Konzepte anpassen. Aber Ist halt auch arbeit :/

Reply:
Ich wüsste nicht wie es nicht bei den Slaves ankommen könnte, da die selects hinzugefügt werden bevor sie in Master und Slaves registriert werden.

Reply:
Aber gut, habe ich entfernt